fluentd/README.md

```diff
@@ -20,3 +20,18 @@ is already created by ansible installer.
 Fluentd pod on startup automatically determines from the `node-config.yaml`
 whether to setup `in_tail` plugin to parse cri-o formatted logs in
 `/var/log/containers/*` or whether to read logs from docker driver.
+
+## Utilities
+### sanitize_msg_chunks
+Sanitize file buffer chunks by removing corrupt records.
+
+There are known [cases](https://bugzilla.redhat.com/show_bug.cgi?id=1562004) where fluentd is stuck processing
+messages that were buffered to corrupt file buffer chunks. This utility is run manually and deserializes each
+file chunk, perform a limited set of operations to confirm message validity. Use this utility by:
+
+* Stopping fluentd
```

Comment:
When you say to stop fluentd you mean rsh into the pod and then kill the running process? Doesn't the pod go away when we do this?

Reply:
to stop fluentd on a given node - use `oc label node $node logging-infra-fluentd-`, then use `oc get pods` or `ps` to make sure fluentd is not running.

Reply:
So we aren't running this utility from within the fluentd pod?